src/Microsoft.Health.Dicom.Blob/Registration/DicomServerBuilderBlobRegistrationExtensions.cs

```diff
@@ -17,59 +17,113 @@ namespace Microsoft.Extensions.DependencyInjection
 {
     public static class DicomServerBuilderBlobRegistrationExtensions
     {
-        private const string DicomServerBlobConfigurationSectionName = "DicomWeb:DicomStore";
-
         /// <summary>
-        /// Adds the blob data store for the DICOM server.
+        /// Adds the data store for the DICOM server.
         /// </summary>
-        /// <param name="serverBuilder">The DICOM server builder instance.</param>
+        /// <param name="serverBuilder">The DICOM server builder.</param>
         /// <param name="configuration">The configuration for the server.</param>
-        /// <returns>The server builder.</returns>
-        public static IDicomServerBuilder AddBlobStorageDataStore(this IDicomServerBuilder serverBuilder, IConfiguration configuration)
+        /// <returns>The service collection.</returns>
```

Comment:
Looks like we are still returning a serverBuilder, not service collection

Reply:
Done.